test-utils/src/main/groovy/ch/tutteli/gradle/test/SettingsExtension.groovy

```diff
@@ -67,7 +82,7 @@ class SettingsExtension implements ParameterResolver, AfterEachCallback {
     void afterEach(ExtensionContext context) throws Exception {
         SettingsExtensionObject settingsSetup = getStore(context).get("settingsSetup") as SettingsExtensionObject
         if (settingsSetup != null) {
-            deleteTmp(settingsSetup.tmpPath)
+            // deleteTmp(settingsSetup.tmpPath)
```

Comment:
what is the reason that you commented it out?

Reply:
I needed to debug the failing spek v1 test and wanted to look at the test report. Forgot to comment it in before commiting.

Done.